<!-- Launches obstacle detector with default values of parameters. -->

<launch>
  <node name="obstacle_detector" pkg="obstacle_detector" type="obstacle_detector" respawn="false" required="false" output="screen">
    <param name="frame_id"       type="string" value="base"/>
    <param name="scan_topic"     type="string" value="front_scan"/>
    <param name="pcl_topic"      type="string" value="pcl"/>
    <param name="obstacle_topic" type="string" value="obstacles"/>
    <param name="marker_topic"   type="string" value="obstacle_markers"/>

    <param name="use_scan"            type="bool" value="true"/>
    <param name="use_pcl"             type="bool" value="false"/>
    <param name="publish_markers"     type="bool" value="true"/>
    <param name="save_snapshot"       type="bool" value="false"/>
    <param name="use_split_and_merge" type="bool" value="false"/>

    <param name="min_group_points"    value="3"/>
    <param name="max_group_distance"  value="0.100"/>
    <param name="distance_proportion" value="0.006136"/>
    <param name="max_split_distance"  value="0.100"/>

    <param name="max_merge_separation" value="0.200"/>
    <param name="max_merge_spread"     value="0.070"/>
    <param name="max_circle_radius"    value="0.300"/>
    <param name="radius_enlargement"   value="0.050"/>
  </node>
</launch>

